ru/validation.php

```diff
@@ -15,26 +15,26 @@
     */
 
     "accepted"             => "Вы должны принять :attribute.",
-    "active_url"           => "Поле :attribute недействительный URL.",
-    "after"                => "В поле :attribute должна быть дата после :date.",
+    "active_url"           => "Поле :attribute содержит недействительный URL.",
+    "after"                => "Поле :attribute должно содержать дату не ранее :date.",
```

Comment:
The original say **after** but you write **not earlier than**

General comment: please no bikeshedding. Ether fix errors or translate which is not translated yet.